askama_derive/src/generator.rs

```diff
@@ -1273,29 +1273,24 @@ impl<'a> Generator<'a> {
         if name == "json" {
             return Err("the `json` filter requires the `serde-json` feature to be enabled".into());
         }
-        #[cfg(not(feature = "serde-yaml"))]
-        if name == "yaml" {
-            return Err("the `yaml` filter requires the `serde-yaml` feature to be enabled".into());
-        }
 
-        const FILTERS: [&str; 2] = ["safe", "yaml"];
-        if FILTERS.contains(&name) {
+        let display_wrap = if name == "safe" {
             buf.write(&format!(
                 "{CRATE}::filters::{}({}, ",
                 name, self.input.escaper
             ));
+            DisplayWrap::Wrapped
         } else if crate::BUILT_IN_FILTERS.contains(&name) {
             buf.write(&format!("{CRATE}::filters::{name}("));
+            DisplayWrap::Unwrapped
         } else {
             buf.write(&format!("filters::{name}("));
-        }
+            DisplayWrap::Unwrapped
+        };
 
         self._visit_args(buf, args)?;
         buf.write(")?");
-        Ok(match FILTERS.contains(&name) {
-            true => DisplayWrap::Wrapped,
-            false => DisplayWrap::Unwrapped,
-        })
```

Comment:
Thanks for that change! Not searching the list twice is such a low-hanging fruit that I wonder why no one noticed that before. :)